routes/models/specification.js

```diff
@@ -0,0 +1,53 @@
+var md5= require('md5') ;
+var DB = require('../config.js') ;
+var random = require('../../libs/random.js') ;
+var Boom =  require('boom');
+
+module.exports.add = function(req,res) {
+    var specification =  req.payload ;
+    specification.created_by = new Date();
+    DB.conn.queryAsync("INSERT INTO "+DB.specifications +" SET ?" , specification ).then(function(result) {
```

Comment:
@tusharncuelogic @sandeepp-cuelogic @gauravc-cuelogic @pritig-cuelogic exception handling should be there.